content/strategy-goals/strategy/code-graph/search/product.md

```diff
@@ -12,26 +12,33 @@ Quicklinks:
 
 ## Where we are now
 
-Sourcegraph is the leading code search product in the market, used by tens of thousands of developers across enterprises of all sizes. It has advanced, novel search capabilities, such as [structural search](https://learn.sourcegraph.com/how-to-use-structural-search-in-sourcegraph) and [diff/commit search](https://docs.sourcegraph.com/code_search/explanations/features#commit-diff-search), as well as capabilities that extend search, such as [Code monitors](https://docs.sourcegraph.com/code_monitoring). Our work-in-progress includes supporting other use cases powered by search, such as onboarding new developers, and integrations with Sourcegraph.
+Sourcegraph is the leading code search product in the market, used by tens of thousands of developers across enterprises of all sizes. It has advanced, novel search capabilities, such as [structural search](https://learn.sourcegraph.com/how-to-use-structural-search-in-sourcegraph) and [diff/commit search](https://sourcegraph.com/notebooks/Tm90ZWJvb2s6MTI=), as well as capabilities that extend search, such as [Code monitors](https://docs.sourcegraph.com/code_monitoring).
+
+<!-- TODO: fix up docs link to go to Notebooks when docs PR is merged -->
+
+We've launched [Notebooks](https://sourcegraph.com/notebooks?tab=explore) in public beta. You can create and share Notebooks with a free Sourcegraph cloud account. Read more about Notebooks in the [docs](https://docs.sourcegraph.com). This quarter will also see a [Code monitors](https://sourcegraph.com/code-monitoring?visible=4) release featuring a 10,000 repository limit on monitors (a 200x increase), major peformance improvements, multiple actions per monitor, Slack integration, and webhook support.
 
 ## What's next and why
 
-### FY22Q4
+### FY23Q1
```

Comment:
No need to call out FY23Q1 as we can let the reader assume these are current, and there aren't any other quarters listed with goals to differentiate this section from.